chia/full_node/mempool_manager.py

```diff
@@ -131,11 +133,13 @@ def __init__(
 
         # The mempool will correspond to a certain peak
         self.peak: Optional[BlockRecord] = None
-        self.mempool: Mempool = Mempool(
-            self.mempool_max_total_cost,
-            uint64(self.nonzero_fee_minimum_fpc),
-            uint64(self.max_block_clvm_cost),
+        self.fee_estimator: FeeEstimatorInterface = create_bitcoin_fee_estimator(self.max_block_clvm_cost)
+        mempool_info = MempoolInfo(
+            CLVMCost(uint64(self.mempool_max_total_cost)),
+            FeeRate(uint64(self.nonzero_fee_minimum_fpc)),
+            CLVMCost(uint64(self.max_block_clvm_cost)),
         )
+        self.mempool: Mempool = Mempool(mempool_info, self.fee_estimator)
```

Comment:
`self.mempool: Mempool = Mempool(mempool_info, self.fee_estimator)`

This line is the fix.